### PR TITLE
Contribute docs

### PR DESCRIPTION
**Pull request by migara**
## Description

Add CONTRIBUTING.md to terraform-best-practices repo

## Motivation and Context

We have six Terraform repos and maintaining CONTRIBUTING.md makes more sense

## How Has This Been Tested?

N/A

## Types of changes

- Documentation

N/A

### CODE REVIEW

**Review by devsecfranklin — APPROVED** [review]
lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+### New Contributors
+
+1. Search the [issues](https://github.com/PaloAltoNetworks/terraform-aws-vmseries-modules.git/issues) to see if there is an existing issue. If not, open an issue (note the issue ID).
+
+1. Fork the repository to your personal namespace (only need to do this once).
+
+1. Clone the repo from your personal namespace.
+
+   `git clone https://github.com/{username}/tterraform-aws-vmseries-modules.git`
```
Comment (devsecfranklin) [inline CONTRIBUTING.md:25]: There is a type here, `tteraform` with two T's

**Review by devsecfranklin — CHANGES_REQUESTED** [review]
one small typo found

File: CONTRIBUTING.md (hunk 1 above)
Comment (migara) [reply CONTRIBUTING.md:25]:
```suggestion
   `git clone https://github.com/{username}/terraform-aws-vmseries-modules.git`
```

**Pull request merged**